runhouse/resources/hardware/cluster.py

```diff
@@ -480,7 +482,13 @@ def _start_server_cmds(cls, restart, restart_ray, screen, create_logfile):
                 Path(cls.SERVER_LOGFILE).touch()
             cmds.append(cls.START_SCREEN_CMD)
         else:
-            cmds.append(cls.SERVER_START_CMD)
+            server_start_cmd = cls.SERVER_START_CMD
+            if custom_server_host is not None:
+                server_start_cmd += f" --host {custom_server_host}"
```

Comment:
I'm a bit confused - above it looks like this would need to be `--custom_server_host` but here it's just `--host`?

Reply:
Oh oh http_server.py takes `--host`, not the CLI, got it.

Reply:
Yep! Renamed all to host.